Modix.Data/Models/Promotions/PromotionCommentSearchCriteria.cs

```diff
@@ -52,16 +52,16 @@ public static IQueryable<PromotionCommentEntity> FilterBy(this IQueryable<Promot
 
             return query
                 .FilterBy(
-                    x => x.CampaignId == criteria.CampaignId.Value,
+                    x => x.CampaignId == criteria.CampaignId!.Value,
                     !(criteria.CampaignId is null))
                 .FilterBy(
-                    x => x.Sentiment == criteria.Sentiment.Value,
+                    x => x.Sentiment == criteria.Sentiment!.Value,
                     !(criteria.Sentiment is null))
                 .FilterBy(
-                    x => x.CreateAction.Created >= criteria.CreatedRange.Value.From.Value,
+                    x => x.CreateAction.Created >= criteria.CreatedRange!.Value.From!.Value,
```

Comment:
Same kinda expression as above. If we decide to keep that one changed, this should change with it.